### PR TITLE
Aggregate Brazil

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Description
Aggregates Brazil and removes the unused BR-I zone as it has no geometry, no parser and no useful config information.

>**Note**
> Timezone information is missing from this PR, but all subzones have `timezone: Null` so it shouldn't make a difference. I'd be happy to include it if we know which timezone to use though (Brazil has 3 or 4 depending on daylight savings).

>**Warning**
>This should not be merged until #4677 is resolved.

### CODE REVIEW

File: config/exchanges/AR_BR.yaml
Comment (unitrium) [inline config/exchanges/AR_BR.yaml:1]: Actually I don't think those are necessary.
The aggregation script will keep the exchanges that are need and set them to the parent level when the country level view is selected.

**Review by unitrium — CHANGES_REQUESTED** [review]
Let's restart this now that the US aggregation has been fulfilled :)

I don't think the exchanges are necessary here. And it's probably good to rerun update world.

File: config/exchanges/AR_BR.yaml
Comment (VIKTORVAV99) [reply config/exchanges/AR_BR.yaml:1]: Are you sure? We manually added the others (and I added the ones of the US).

**Comment by VIKTORVAV99** [comment]
> Let's restart this now that the US aggregation has been fulfilled :)
> 
> I don't think the exchanges are necessary here. And it's probably good to rerun update world.

Isn't this still technically blocked by #4677?

File: config/exchanges/AR_BR.yaml
Comment (unitrium) [reply config/exchanges/AR_BR.yaml:1]: Actually yes you're right, I read too fast through the config files. So no changes need here :)

**Comment by VIKTORVAV99** [comment]
> We have a fix on the backend side almost ready to be merged, we might have the time to get it merged next week :)

I'll try and get the merge conflicts cleaned up this weekend then, should be pretty straight forward. 👍🏼

**Comment by unitrium** [comment]
Yes, don't worry yourself for it :)

File: .prettierignore
Comment (VIKTORVAV99) [inline .prettierignore:4]: This was needed for lint staged to pass as prettier could not infer a parser for it and since lint staged explicitly pass files to prettier it failed and I could not commit without it.

File: web/public/locales/de.json
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1426]: Apparently there was a duplicate here, removed it when I fixed the merge conflicts.

**Comment by VIKTORVAV99** [comment]
I noticed there are a few changed that slipped through the merge conflict that shouldn't have so I'll have to take care of those before this is merged.

**Comment by VIKTORVAV99** [comment]
Should be all good to go now.

**Review by unitrium — APPROVED** [review]
Looks good, I will let you know once the fix is merged in the backend side :)

**Comment by unitrium** [comment]
Just merged the latest changes :)

**Comment by VIKTORVAV99** [comment]
Merging this now as the upstream issues blocking it has been resolved! 🎉

**Pull request merged**
